src/main/haskell/kore/src/Kore/Attribute/Exit.hs

```diff
@@ -0,0 +1,72 @@
+{-|
+Module      : Kore.Attribute.Exit
+Description : Exit symbol attribute
+Copyright   : (c) Runtime Verification, 2018
+License     : NCSA
+Maintainer  : thomas.tuegel@runtimeverification.com
+
+-}
+module Kore.Attribute.Exit
+    ( Exit (..)
+    , exitId, exitSymbol, exitAttribute
+    ) where
+
+import           Control.DeepSeq
+                 ( NFData )
+import qualified Control.Monad as Monad
+import           Data.Default
+import           GHC.Generics
+                 ( Generic )
+
+import           Kore.AST.Kore
+import           Kore.Attribute.Parser
+                 ( ParseAttributes (..) )
+import qualified Kore.Attribute.Parser as Parser
+
+-- | @Exit@ represents the @exit@ attribute for symbols.
+newtype Exit = Exit { isDeclaredExit :: Bool }
```

Comment:
The field can just be `isExit`.
```suggestion
newtype Exit = Exit { isExit :: Bool }
```
The distinction between `isFoo` and `isDeclaredFoo` is important for properties like functionality; a symbol may be declared functional, but the pattern is only functional if its arguments are also functional.